src/python/grpcio/grpc/experimental/aio/_channel.py

```diff
@@ -77,7 +80,7 @@ def __call__(self,
             raise NotImplementedError("TODO: metadata not implemented yet")
 
         if credentials:
-            raise NotImplementedError("TODO: credentials not implemented yet")
+            credentials = credentials._credentials
```

Comment:
1. Let's use `is not None` to check if credentials exist;
2. Let's use a different variable name to reduce confusion.